lib/smart_proxy_dns_infoblox/dns_infoblox_main.rb

```diff
@@ -75,6 +75,20 @@ def ib_delete(clazz, params)
 
       raise Proxy::Dns::NotFound, "Cannot find #{clazz.class.name} entry for #{params}" if record.nil?
       record.delete || (raise Proxy::Dns::NotFound, "Cannot find #{clazz.class.name} entry for #{params}")
+
+      ib_clear_dns_cache(record)
```

Comment:
Return value could matter, I am not sure here. Just return `record` as the last statement perhaps? This can return `false` when WAPI is too old which could possibly break things.